tests/performance/test_data_analysis/unit_tests/test_data_loader.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""Unit tests for the DataLoader class"""
+
+import pandas as pd
+
+from pytest_mock import MockerFixture
+from statistical_data_analyzer import DataLoader
+
+
+def test_dataloader_initialization(mocker: MockerFixture) -> None:
+    """Test that verifies that the class is initialized correctly and that the
```

Comment:
Same here, the first line should be a summary. Detailed information should be in a description section, separated by one line

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5622/commits/a8be0593badbbd04bb3211468f64ad2689d17bf3